packages/in_app_purchase/in_app_purchase_ios/lib/src/store_kit_wrappers/enum_converters.dart

```diff
@@ -40,6 +41,10 @@ class SKTransactionStatusConverter
       case SKPaymentTransactionStateWrapper.restored:
         return PurchaseStatus.restored;
       case SKPaymentTransactionStateWrapper.failed:
+        if (error != null && (error.code == 2 || error.code == 15)) {
+          return PurchaseStatus.canceled;
```

Comment:
Typo in the word `cancelled`. Should be with double "l":
```suggestion
          return PurchaseStatus.cancelled;
```